src/borough/borough.controller.ts

```diff
@@ -50,6 +52,21 @@ export class BoroughController {
     );
   }
 
+  @UsePipes(
+    new ZodTransformPipe(
+      findCommunityDistrictGeoJsonByBoroughIdCommunityDistrictIdPathParamsSchema,
+    ),
+  )
+  @Get("/:boroughId/community-districts/:communityDistrictId/geojson")
+  async findGeoJsonById(
```

Comment:
Whoops, this is a copy and paste mistake. I borrowed much of the code from the city council district geojson implementation, which could abbreviate because of the "in domain" rule. But you are correct, that rule does not apply to getting Community Districts while in Boroughs. I will adjust.